naming_conventions.md

```diff
@@ -94,81 +72,84 @@ Where `<WHAT>` is one of the following:
 
 Examples:
 
--   **activity\_main**: content view of the MainActivity
--   **fragment\_articledetail**: view for the ArticleDetailFragment
--   **view\_menu**: layout inflated by custom view class MenuView
--   **item\_article**: list item in ArticleRecyclerView
--   **layout\_actionbar\_backbutton**: layout for an actionbar with a
-    backbutton (too simple to be a customview)
+-  **activity\_main**: content view of the MainActivity
+-  **fragment\_articledetail**: view for the ArticleDetailFragment
+-  **view\_menu**: layout inflated by custom view class MenuView
+-  **item\_article**: list item in ArticleRecyclerView
+-  **layout\_footer**: layout for a footer included in several layouts throughout the app
 
 
 Strings
 -------
+Strings are accessed using `R.string`, so `<WHAT>` is irrelevant. Either we use `<WHERE>` to indicate 
+where the string will be used:
 
-The `<WHAT>` part for Strings is irrelevant. So either we use `<WHERE>`
-to indicate where the string will be used:
-
-#`<WHERE>_<DESCRIPTION>`
+### `<WHERE>_<DESCRIPTION>`
 
 or `all` if the string is reused throughout the app:
 
-#`all_<DESCRIPTION>`
+### `all_<DESCRIPTION>`
 
 Examples:
 
--   **articledetail\_title**: title of ArticleDetailFragment
--   **feedback\_explanation**: feedback explanation in FeedbackFragment
--   **feedback\_namehint**: hint of name field in FeedbackFragment
--   **all\_done**: generic "done" string
+-  **articledetail\_title**: title of ArticleDetailFragment
+-  **feedback\_explanation**: explanation in FeedbackFragment
+-  **all\_done**: generic "done" string
 
-`<WHERE>` obviously is the same for all resources in the same view.
+`<DESCRIPTION>` can be split into multiple parts for improved heirarchy. For example, in a form capturing
+information about an employee, you could have: 
 
+-  **employeeform\_personal\_name**: name of individual 
+-  **employeeform\_personal\_address**: address of individual
+-  **employeeform\_company\_name**: name of company
```

Comment:
Should be made clearer that "personal" and "company" are different sections on the form.